redirects.yaml

```diff
@@ -314,87 +261,47 @@ community/ubuntuvalues/?: "/about/about-ubuntu/diversity"
 communitysupport/webforums/?: "https://community.ubuntu.com/t/ubuntu-forums/22"
 contact-us/?: "/about/contact-us"
 contact/CDDistributor/?: "/download/desktop"
-contact/marketplace/?: "http://partners.ubuntu.com"
+contact/marketplace/?: "https://partners.ubuntu.com"
 contact/trademark/?: "/legal/terms-and-policies/contact-us"
 contact/trademarkviolation/?: "/legal/terms-and-policies/contact-us"
 contact/training/?: "/cloud/contact-us?product=cloud-training"
 contact/whitepaper-thankyou/?: "https://insights.ubuntu.com/white-papers/"
 contact/whitepaper/?: "https://insights.ubuntu.com/white-papers/"
 containers/kubernetes/?: "/kubernetes"
-db2/?: "http://partners.ubuntu.com/ibm"
+db2/?: "https://partners.ubuntu.com/ibm"
 de/?: "/"
 debian/?: "/about/about-ubuntu/ubuntu-and-debian"
-dell/?: "http://partners.ubuntu.com/dell"
-dell/uec/?: "http://partners.ubuntu.com/dell"
+dell/?: "https://partners.ubuntu.com/dell"
+dell/uec/?: "https://partners.ubuntu.com/dell"
 desktop/business/?: "/desktop/enterprise"
 desktop/dev/?: "/desktop/developers"
 desktop/developers/thin-client/?: "/desktop"
-desktop/features/games-apps/?: "/desktop/features"
-desktop/features/more-apps/?: "/desktop/features"
-desktop/features/music-and-mobile/?: "/desktop/features"
```

Comment:
0.0.0.0:8001/desktop/features/music-and-mobile/ redirects to 0.0.0.0:8001/desktop/features/music-and-mobile which 404s

Reply:
again, no redirect, but I have added one.